src/GeneticSharp.Domain/OperatorsStrategies/TplOperatorsStrategy.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GeneticSharp.Domain.Chromosomes;
+using GeneticSharp.Domain.Crossovers;
+using GeneticSharp.Domain.Mutations;
+using GeneticSharp.Domain.Populations;
+using GeneticSharp.Domain.Randomizations;
+
+namespace GeneticSharp.Domain
+{
+    public class TplOperatorsStrategy : IOperatorsStrategy
```

Comment:
Please, add documentation to the class explaining its purpose.

Reply:
Done